src/test/java/org/snomed/snowstorm/core/data/services/BranchMergeServiceTest.java

```diff
@@ -2109,6 +2116,824 @@ void testAutoMergeWhenMemberDeletedSomewhereAndVersionedElsewhere() throws Servi
 		assertVersioned(referenceSetMember, 20210812);
 	}
 
+	/*
+	 * The Extension upgrades to the latest version of International. After doing the upgrade, the Extension's task then
+	 * does a rebase, which results in no conflicts as the task has no authoring changes. The latest and greatest components
+	 * are present on the task.
+	 * */
+	@Test
+	void testUpgradeExtensionWhenExtensionTaskHasNoAuthoringChanges() throws ServiceException, InterruptedException {
```

Comment:
A number of tests have been added which cover our 4 scenarios we spoke about privately @CoderMChu